database/factories/AnimeFactory.php

```diff
@@ -0,0 +1,105 @@
+<?php
+namespace Database\Factories;
+
+use App\CarbonDateRange;
+use App\Anime;
+use App\Testing\JikanDataGenerator;
+use MongoDB\BSON\UTCDateTime;
+
+
+class AnimeFactory extends JikanModelFactory
+{
+    use JikanDataGenerator;
+
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Anime::class;
+
+
+    protected function definitionInternal(): array
+    {
+        $mal_id = $this->createMalId();
+        $title = $this->createTitle();
+        $status = $this->faker->randomElement(["Currently Airing", "Completed", "Upcoming"]);
+        [$aired_from, $aired_to] = $this->createActiveDateRange($status, "Currently Airing");
+
+        return [
```

Comment:
From the factories or from where?

Reply:
@pushrbx Yep the factories. The Character factory creates a `images` property where the Anime/Manga factory does not. Is this intended behavior?

Reply:
It can be added, but so far the efforts for this were focused on having mock data for filtering. None of the filtering features do anything with the "images" attribute. Not sure if we ever going to have test for that specific attribute. So I've considered it as optional in case of Anime/Manga, but in case of Character I've added it for the sake of completeness, nothing else.

I will add an empty array regardless. Also for context: think of these as some sort of set of tools to create repeatable/replicable dummy data.

Reply:
@pushrbx Yep, that's why I was asking whether we could remove it altogether since it won't be queried in the tests.

Reply:
👍 done